search2/requirements.txt

```diff
@@ -1,8 +1,9 @@
-maturin==0.14.13
-ray==2.3.0
--f https://storage.googleapis.com/jax-releases/jax_cuda_releases.html
-jax[cuda]==0.4.4
-pyarrow==11.0.0
+maturin==0.15.1
+ray==2.4.0
```

Comment:
jax, numpyro will likely be used for performing inference during search and indexing (e.g making vector embeddings, filtering rows based on the results of a statistical test, as seen in SeqFromQuery.pm. By having these in now, we can build a Docker image that can be used for CI/CD. We can also hold off until the modules that need these are built, your call.

Reply:
I'm probably missing something here-- can we not build the CI/CD image without them if they're not currently required?  My priorities would be:

1. Having CI up and running
2. Having the requirements.txt faithfully reflect the current requirements.